src/colour/web/index.md

```diff
@@ -120,4 +120,4 @@ Use for success messages
 Indicative examples for illustrative purposes only.
 {% endcallout %}
 
-![TODO](./example.png)
+![A screenshot of the GOV.UK homepage on desktop, showing the colours of the logo, header and links, both visited and unvisited.](./example.png)
```

Comment:
(Optional) It might be worth mentioning what the colours are as it's an example of how colours are used.